roles/node/templates/testnet/config.toml.j2

```diff
@@ -0,0 +1,125 @@
+wasm_dir = "{{ node_wasm_dir }}"
+
+[ledger]
+genesis_time = "2024-12-05T23:07:03.000000000+00:00"
+chain_id = "{{ namada[type].chain_id }}"
+
+[ledger.shell]
+base_dir = "{{ node_network_dir }}"
+storage_read_past_height_limit = 3600
+db_dir = "{{ node_data_dir }}"
+cometbft_dir = "{{ node_cometbft_dir }}"
+tendermint_mode = "{{ 'Full' if type == 'fullnode' else 'Validator' }}"
+
+[ledger.cometbft]
+proxy_app = "{{ node_type_settings.proxy_app.address.default }}:{{ namada[type].ports.proxy_port }}"
+moniker = "{{ org }}"
+fast_sync = true
+db_backend = "goleveldb"
+db_dir = "{{ node_cometbft_data_dir }}"
+log_level = "info"
+log_format = "plain"
+genesis_file = "{{ node_cometbft_config_dir }}/genesis.json"
+priv_validator_key_file = "{{ node_cometbft_config_dir }}/priv_validator_key.json"
```

Comment:
we should already have variables for priv_validator_key_file path and priv_validator_state_file path, if yes lets use them